### DISCUSSION TITLE
I use @walletconnect/ethereum-provider to connect to the zerion wallet, but there will be a pairing exsit error prompt, and I don’t know how to solve it

### DISCUSSION THREAD

**Discussion by lrlazy** [WalletConnect/walletconnect-monorepo#3069] [Developer Support]
I use @walletconnect/ethereum-provider to connect to the zerion wallet, but there will be a pairing exsit error prompt, and I don’t know how to solve it

**Comment by crypblizz8** [comment]
Can you provide more context on your setup / package vesrion + running code?
